rskj-core/src/test/java/co/rsk/peg/BridgeSupportReleaseBtcTest.java

```diff
@@ -243,7 +243,7 @@ void handmade_release_after_rskip_146_185_before_rskip326() throws IOException {
     }
 
     @Test
-    void handmade_release_after_rskip_146_185_326_before_rskip428() throws IOException {
+    void release_after_rskip_146_185_326_before_rskip_427_428_no_logReleaseBtcRequestReceived_value_in_satoshis() throws IOException {
```

Comment:
```suggestion
    void release_after_rskip_326_before_rskip_427_no_logReleaseBtcRequestReceived_value_in_satoshis() throws IOException {
```

same here?

also, something like
```suggestion
    void logReleaseBtcRequestReceived_after_rskip326_before_rskip427_shouldLogValueInSatoshis() throws IOException {
```
is clearer?